test/docker-compose.yaml

```
@@ -0,0 +1,30 @@
+services:
+  db:
+    build:
+      context: ./postgis_docker
+      dockerfile: ./Dockerfile
+    volumes:
+      - ./postgis_docker/setupdata.sql:/docker-entrypoint-initdb.d/setup-db.sql
+    environment:
+      - POSTGRES_DB=gis
+      - POSTGRES_USER=docker
+      - POSTGRES_PASS=docker
+      - ALLOW_IP_RANGE=0.0.0.0/0
+      - POSTGRES_MULTIPLE_EXTENSIONS=postgis,hstore,postgis_topology,postgis_raster,pgrouting
```

```suggestion
      - POSTGRES_MULTIPLE_EXTENSIONS=postgis
```

will that start faster?